typescript/infra/helm/offchain-lookup-server/templates/deployment.yaml

```
@@ -0,0 +1,52 @@
+            periodSeconds: 10
+          imagePullPolicy: IfNotPresent
+          command: ["yarn"]
+          args: ["--cwd", "typescript/ccip-server", "run", "start"]
+          ports:
+            - name: http
+              containerPort: {{ .Values.port }}
+            - name: metrics
+              containerPort: {{ .Values.prometheus_port }}
+          env:
+            {{- range .Values.secrets.secret_envs }}
+            - name: {{ . }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)